bin/oio-blob-mover

```diff
@@ -16,12 +16,99 @@
+    parser.add_argument('--daemon', action='store_true',
+                        help="Run mover as a daemon.")
+    parser.add_argument('--report-interval', type=int,
+                        help="Interval between passes in seconds")
+    parser.add_argument('--usage-check-interval', type=int,
+                        help="Interval between disk usage check in seconds")
+    parser.add_argument('--verbose', '-v', action='store_true',
+                        help="More verbose output")
+    parser.add_argument('--user', help='run daemon as user')
+    parser.add_argument('--bytes-per-second', type=int,
+                        help="Throttle: max byte per second")
+    parser.add_argument('--chunks-per-second', type=int,
+                        help="Throttle: max chunks per second")
+    levels = ['DEBUG', 'INFO', 'WARN', 'ERROR']
+    parser.add_argument('--log-level', choices=levels,
+                        help="Log level")
+    parser.add_argument('--log-syslog-prefix', help="Syslog prefix")
+    parser.add_argument('--log-facility', help="Log facility")
+    parser.add_argument('--log-address', help="Log address")
+
+    return parser
+
+
+def generate_config_file(path, mode):
+    args = make_arg_parser().parse_args()
+
+    def add_value(dic, key, value):
+        if value is not None:
+            dic[key] = value
+
+    def dic_to_string(dic, header):
+        for key in dic:
+            header += key + " = " + str(dic[key]) + "\n"
+        return header
+
+    def create_content():
+        header = "\n[blob-mover]\n"
+        cont = dict()
+        add_value(cont, 'namespace', args.namespace)
+        add_value(cont, 'volume', args.volume)
+        add_value(cont, 'log_level', args.log_level)
+        add_value(cont, 'log_facility', args.log_facility)
+        add_value(cont, 'log_address', args.log_address)
+        add_value(cont, 'syslog_prefix', args.log_syslog_prefix)
+        add_value(cont, 'usage_target', args.usage_target)
+        add_value(cont, 'usage_check_interval',
+                  args.usage_check_interval)
+        add_value(cont, 'report_interval', args.report_interval)
+        add_value(cont, 'bytes_per_second', args.bytes_per_second)
+        add_value(cont, 'chunks_per_second', args.chunks_per_second)
+        add_value(cont, 'user', args.user)
+        return dic_to_string(cont, header)
+
+    with open(path, mode) as conf:
+        cont = create_content()
+        conf.write(cont)
+    return path
+
+
+def main():
+    args = make_arg_parser().parse_args()
+    limit = args.limit
+    usage_target = args.usage_target
+    daemon = args.daemon
+    verbose = args.verbose
+    config = args.config
+    if args.generate_config:
+        config = generate_config_file(config, 'w')
+    if args.edit_config:
+        config = generate_config_file(config, 'a')
+    run_daemon(BlobMover, config, limit=limit,
+               usage_target=usage_target, daemon=daemon,
+               verbose=verbose)
```

Comment:
`limit` and` usage_target` should be added to the `conf`. That way, you only manage the configuration in the constructor in `BlobMoverWorker`.